src/pages/data-reporting/reports-api/index.mdx

```diff
@@ -520,7 +520,9 @@ https://api.livechatinc.com/v3.4/reports/chats/engagement \
 
 ### Greetings Conversion
 
-Shows the number of greetings sent to the customers and how many of those resulted in a chat or a goal. Greetings are also known as targeted messages.
+Shows the number of greetings sent to the customers and how many of those resulted in a chat or a goal. Greetings are also known as targeted messages. 
+
+The method will return only up to 100 values for each day, and it's connected with the response size. If there are more than 100 values, please use the Agent App to access the full data for a given day.
```

Comment:
> and it's connected with the response size

I think it's unnecessary. We don't need to explain why our responses have a particular shape.

Reply:
Yes, you are right. There is no need for that.